lib/api/tests/js_instance.rs

```diff
@@ -511,12 +511,12 @@ mod js {
             args: &[Val],
         ) -> Result<Vec<Val>, RuntimeError> {
             let memory: &Memory = env.data().memory.as_ref().unwrap();
-            let memory_val = memory.uint8view(&env).get_index(0);
+            let memory_val = memory.view(&env).uint8view().get_index(0);
```

Comment:
Let's try to use the MemoryView API directly

Reply:
done